src/test/java/org/folio/rest/impl/PermsAPITest.java

```diff
@@ -95,7 +95,7 @@ public void testUpdateUserPermissionsFail(TestContext context) {
     Context vertxContext = vertx.getOrCreateContext();
     PostgresClient postgresClient = PostgresClient.getInstance(vertxContext.owner(), tenantId);
     postgresClient.startTx(s -> {
-      Future<Void> future = PermsAPI.updateUserPermissions(s, "bad",
+      Future<Void> future = api.updateUserPermissions(s, "bad",
```

Comment:
`updateUserPermissions` is a static method. Consider to use `PermsAPI.updateUserPermissions` but remove the unnecessary line 92 `PermsAPI api = new PermsAPI();`

Reply:
ok